### PR TITLE
docs: Update Readme and generated jazzy documentation

### PR DESCRIPTION
**Pull request by EnriqueL8**
- quick fixes to the README
- `jazzy` tag on PR to generated docs

### CODE REVIEW

File: docs/Classes.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
         <a class="header-link" href="index.html">
           KituraContracts Docs
         </a>
-         (100% documented)
+         (98% documented)
```
Comment (djones6) [inline docs/Classes.html]: :cry:

**Comment by djones6** [comment]
@EnriqueL8 I've enabled the Jazzy generation and we have a docs commit - if you're happy with the result I will merge.

File: docs/Classes.html (hunk 1 above)
Comment (djones6) [reply docs/Classes.html]: Looks like the [Operation protocol](https://github.com/IBM-Swift/KituraContracts/blob/1.0.0/Sources/KituraContracts/Contracts.swift#L1097) is missing documentation on its functions - could we add these as part of this PR?

Comment (EnriqueL8) [reply docs/Classes.html]: done

File: Sources/KituraContracts/Contracts.swift [hunk 2]
```diff
@@ -1095,8 +1095,13 @@ public enum Operator: String, Codable {
   An identifier for an operation object.
 */
 public protocol Operation: Codable {
+  /// Createa an Operation from a string value
   init(string: String) throws
+
+  /// Returns the string value associated with an Operation
```
Comment (djones6) [inline Sources/KituraContracts/Contracts.swift]: Could we be more descriptive? ie. what does this string represent?

File: Sources/KituraContracts/Contracts.swift [hunk 3]
```diff
@@ -1098,12 +1098,18 @@ public protocol Operation: Codable {
...
   /// ```swift
   ///  let range = InclusiveRange(start: 5, end: 10)
   /// ```
   /// would be represented as `"5,10"`, which in the URL would translate to: `?range=5,10`
+  /// This URL format is not an API but an implementation detail that could change in the future.
+  /// The URL doesn't encode the operator itself instead it is inferred at
+  /// decoding time by the type information associated with that key.
+  /// The type information used to decode this URL format is defined by
+  /// the QueryParams structure associated with a route.
+  /// The key name in the url maps to the field name in the QueryParams structure.
```
Comment (djones6) [inline Sources/KituraContracts/Contracts.swift:1112]: :+1: 
Thanks!

**Pull request merged**